libs/libcommon/src/libcommon/queue.py

```diff
@@ -96,6 +96,18 @@ class JobDoesNotExistError(DoesNotExist):
     pass
 
 
+class AlreadyStartedError(Exception):
```

Comment:
```suggestion
class JobAlreadyStartedError(Exception):
```
Or something like that?